SharpBucket/V2/Pocos/Issue.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+
+namespace SharpBucket.V2.Pocos
+{
+    public class Issue
+    {
+        public int id { get; set; }
+
+        public IssuePriority priority { get; set; }
+
+        public IssueKind kind { get; set; }
+
+        public Repository repository { get; set; }
```

Comment:
Probably more a `RepositoryInfo` than a full `Repository` here.

```suggestion
        public RepositoryInfo repository { get; set; }
```